minecraft_servers/NinjaMC/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "server_name": "NinjaMC",
+  "nice_name": "NinjaMC.net",
+  "direct_ip": "NinjaMC.net",
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "Discord": "discord.gg/ninjamc"
```

Comment:
JSON conventions, key has to be lowercase

Reply:
I changed it

Reply:
I changed it now